examples/argparse_example4.jl

```diff
@@ -13,13 +13,13 @@ function main(args)
     @add_arg_table s begin
         "--opt1"
             action = :append_const   # appends 'constant' to 'dest_name'
-            arg_type = ByteString    # the only utility of this is restricting the dest array type
+            arg_type = String
```

Comment:
Could you keep the comment here?

Reply:
I created a new PR so not to have two separate commits, I am closing this PR.